index.html

```diff
@@ -48,5 +48,157 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio3.png" alt="Snapshot3">
+                    <div class="description">
+                        <h2 class="workstitle">Multi-Post Stories</h2>
+                        <div class="info">
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+    
+        </div>
+    </section>
+
+    <section class="about-section" id="About">
+        <h1 class="title">About<span class="title">Myself!</span>
+        </h1>
+
+        <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <p class="aboutConnect">LET’S CONNECT</p>
+    
+              <ul class="social-media-icons">
+                <li><a href=""><img class="icon twitter" 
+                    src="src/Twitter.png" alt="twitter-icon"/></a></li>
+
+                <li><a href=""><img class="icon linkedin" 
+                    src="src/Linkedin-icon.png" alt="linkedin-icon"/></a></li>
+
+                <li><a href=""><img class="icon msign" 
+                    src="src/m.png" alt="msign"/></a></li>
+
+                <li><a href=""><img class="icon github" 
+                    src="src/Github.png" alt="github-icon"/></a></li>
+
+                <li><a href=""><img class="icon finger" 
+                    src="src/Angel.png" alt="AngelList"/></a></li>
+              </ul>
+
+            <button type="button" class="resumeButton">Get my resume</button>
+               
```

Comment:
- [ ] The about section includes a list of technologies categories ("Languages", "Frameworks" and "Skills"), which should be in a ```<ul>``` with each of them in a ```<li>```
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ```<ul>``` and each of them a ```<li>```

### Design 
![image](https://user-images.githubusercontent.com/43995830/153704050-ca2421e8-b9e3-44a8-a0eb-a7a7d939e631.png)